pulp_rpm/app/replica.py

```diff
@@ -0,0 +1,40 @@
+from pulpcore.plugin.replica import Replicator
+
+from pulp_glue.rpm.context import (
+    PulpRpmDistributionContext,
+    PulpRpmPublicationContext,
+    PulpRpmRepositoryContext,
+)
+
+
+from pulp_rpm.app.models import RpmDistribution, RpmRemote, RpmRepository
+from pulp_rpm.app.tasks import synchronize as rpm_synchronize
+
+
+class RpmReplicator(Replicator):
+    repository_ctx_cls = PulpRpmRepositoryContext
+    distribution_ctx_cls = PulpRpmDistributionContext
+    publication_ctx_cls = PulpRpmPublicationContext
+    app_label = "rpm"
+    remote_model_cls = RpmRemote
+    repository_model_cls = RpmRepository
+    distribution_model_cls = RpmDistribution
+    distribution_serializer_name = "RpmDistributionSerializer"
+    repository_serializer_name = "RpmRepositorySerializer"
+    remote_serializer_name = "RpmRemoteSerializer"
+    sync_task = rpm_synchronize
+
+    def repository_extra_fields(self, remote):
+        return dict(autopublish=True)
```

Comment:
Is the goal here to have an exact mirror?  In that case we shouldn't use autopublish, we should use `mirror_complete` sync mode.

Reply:
I've performed manual testing and I have not seen any errors. I would like to add tests after domains are enabled for pulp_rpm. The tests would migrate from one domain to another.  The other option is to deploy another Pulp instance to replicate from. Either way, the tests will come in a separate PR.

Reply:
I will update the code to not use autopublish=True.